### ISSUE TITLE
Playground: Ctrl+Left/Right arrow not working

### ISSUE THREAD

**Issue by thany** [microsoft/TypeScript-Website#2954]
**Page URL:** 
https://www.typescriptlang.org/play

**Issue:** 
1. Type any code whatsoever.
2. Use Ctrl+Left/Right arrows to navigate word by word.

It won't do anything. Ctrl+Left/Right doesn't move the caret anywhere, it stays exactly where it is.

Tested on Firefox 118 on Windows 10.

**Comment by Protowalker** [comment]
Can also vouch for this. It works in chrome, and I've tested the same version of monaco that the playground uses and it had no problems in firefox: https://microsoft.github.io/monaco-editor/playground.html?source=v0.32.1#example-creating-the-editor-hello-world

**Comment by DaveCousineau** [comment]
yes, extremely annoying. can't do simple things like ctrl-shift-left to select the previous word. constantly tripping over this whenever I use the editor.

**Comment by thany** [comment]
Wow, how is this basic functionality still not solved?!
